packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/simplifySmalltalkCiLogCheckBoxAction.st

```diff
@@ -0,0 +1,5 @@
+building
+simplifySmalltalkCiLogCheckBoxAction
+	
+	self shouldSimplifySmalltalkCiLog: self shouldSimplifySmalltalkCiLog not.
+	self updateJobLogForId: ( (self jobs at: selectedJob) id )
```

Comment:
```suggestion
some other category not sure
simplifyLogCheckBoxAction
	
	self shouldSimplifyLog: self shouldSimplifyLog not.
	self updateJobLogForId: (self jobs at: selectedJob) id
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code

Idk if the parenthesis are necessary